### PR TITLE
Fixes SEGFAULT in integration_tests/elemental_01.py

### PR DESCRIPTION
**Pull request by Kishan-Ved**
Please check: https://github.com/lcompilers/lpython/pull/2451

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,6 +163,7 @@ Please follow the below steps for Windows:
 
   ```bash
   ctest
+  ./run_tests.py -u
```
Comment (Pranavchiku) [inline README.md:166]: We do `-u` to update test references. To run test simply do `./run_tests.py`

Comment (Kishan-Ved) [reply README.md:166]: The Linux sections didn't mention `./run_tests.py -u` command anywhere, I want to add that. I spent quite a lot of time in figuring that out.

Comment (Kishan-Ved) [reply README.md:166]: Also the macos test doesn't seem to pass, what can be the reason?

Comment (Pranavchiku) [reply README.md:166]: > The Linux sections didn't mention ./run_tests.py -u command anywhere, I want to add that.

Then you may add a new subsection below it mentioning `Update test references` and under it `./run_tests.py -u`

Comment (Pranavchiku) [reply README.md:166]: > what can be the reason?

Looks like some kind of infinite loop ran over, you may need to check if there exists any kind of error.

Comment (Kishan-Ved) [reply README.md:166]: > > what can be the reason?
> 
> Looks like some kind of infinite loop ran over, you may need to check if there exists any kind of error.

In that case, why do other tests pass?

Comment (Kishan-Ved) [reply README.md]: > > The Linux sections didn't mention ./run_tests.py -u command anywhere, I want to add that.
> 
> Then you may add a new subsection below it mentioning `Update test references` and under it `./run_tests.py -u`

Done!

**Review by czgdp1807 — APPROVED** [review]

**Comment by certik** [comment]
@czgdp1807, @Kishan-Ved this PR committed an executable into main. Please be very careful when merging PRs not to add any executables.

**Pull request merged**